src/main/resources/log4j2.xml

```diff
@@ -0,0 +1,23 @@
+<?xnl version="1.0" encoding="UTF-8"?>
+<Configuration status="info">
+<Appenders>
+    <Console name="LogToConsole" target="SYSTEM_OUT">
+        <PatternLayout pattern="%d{HH:mm:ss.SSs} %-5level %logger{36} - %msg%n"/>
+    </Console>
+    <File name="LogToFile" fileName="logs/app.log">
+        <PatternLayout>
+            <Pattern>%d %p %c:%L %m%n</Pattern>
+        </PatternLayout>
+    </File>
+</Appenders>
+<Loggers>
+    <Logger name="mate.academy" level="info" additivity="false">
+        <AppenderRef ref="LogToFile"/>
+        <AppenderRef ref="LogToConsole"/>
+    </Logger>
+    <Root level="error">
+        <AppenderRef ref="LogToFile"/>
+        <AppenderRef ref="LogToConsole"/>
+    </Root>
+</Loggers>
+</Configuration>
```

Comment:
file should end with a new line

Reply:
yea an empty line is here, if they don't It was a mistake